src/gui/qgscollapsiblegroupbox.cpp

```diff
@@ -547,6 +549,9 @@ void QgsCollapsibleGroupBox::showEvent( QShowEvent * event )
 
 QString QgsCollapsibleGroupBox::saveKey() const
 {
+  if ( objectName().isEmpty() || ( mSettingGroup.isEmpty() && window()->objectName().isEmpty() ) )
+    return QString::null;  // cannot get a valid key
```

Comment:
Oops... Ignore me, just getting confused! Anyway, qt docs still say QString::null is evil ;)